### DISCUSSION TITLE
Mounting a host directory onto a VM

### DISCUSSION THREAD

**Discussion by axelniklasson** [macstadium/orka-desktop#9] [General]
Hi gang! I can't seem to figure out how to mount a host directory onto the VM and I can't find it anywhere in the docs. If anyone could point me to the right docs section or let me know how to mount a folder from the host to the VM that would be much appreciated. Thanks!

**Comment by abojak** [comment]
Hi Axel.
Sorry but we have not implemented a GUI SMB feature yet in our application
but a workaround is simply manually enabling a SMB share. Choose System
Settings->General->Sharing->File Sharing on your host or guest.
Then simply open a Finder window on other system and choose Network in the
left pane of Finder to find your new shared file system, connect to it
(login w/ mac user credentials or you can allow guest access when setting
up file sharing), and you can access files, copy/paste etc. between guest
and host that way.
Ex suited to your specific request. Host MacBook name ='myMac' credentials:
"user"/"pass", and the Guest VM name='OrkaTest'. Turn file sharing on your
host. Then on the guest "OrkaTest" VM, look for "myMac" in network and
connect using user/pass credentials of the host:user/pass.
You can also perform the file share enable vice versa (on guest then
connect from host). Sometimes MDM profiles may balk at sharing if Host a
corp machine, so just enable SMB share on VM guest. We have a plan to
implement this through the Orka Desktop GUI soon.
Thanks for the inquiry.
Cheers.

Andrew Bojak
MacStadium Orka PM

**Reply by axelniklasson** [reply]
This worked perfectly, thanks @abojak 🙌
